wire-ios/Wire-iOS/Resources/de.lproj/Localizable.strings

```diff
@@ -2146,7 +2086,7 @@
 "device.details.section.e2ei.status.expired" = "Abgelaufen";
 "device.details.section.e2ei.status.invalid" = "Ungültig";
 "device.details.section.e2ei.serial_number" = "Seriennummer";
-"device.details.section.mls.signature" = "MLS mit Ed25519 Signatur";
+"device.details.section.mls.signature" = "MLS mit %@ Signatur";
```

Comment:
@typfel is it the string you need for the release ?

Reply:
Yes!